src/cpp/tests/testthat/test-rsconnect.R

```diff
@@ -51,6 +51,10 @@ test_that("setting UI prefs updates options", {
 })
 
 test_that(".rs.rsconnectDeployList() includes _quarto.yml and _metadata.yml files (#10995 )", {
+   if (Sys.getenv("QUARTO_ENABLED") == "FALSE") {
```

Comment:
This looks good, but I wonder if it's safer to check if it's `!= "TRUE"` instead of `== "FALSE"`. Equivalent for now, but maybe more in line with how we do IFDEF / IFNDEF